shared/src/main/scala/coop/rchain/store/KeyValueTypedStoreSyntax.scala

```diff
@@ -9,12 +10,25 @@ trait KeyValueTypedStoreSyntax {
   ): KeyValueTypedStoreOps[F, K, V] = new KeyValueTypedStoreOps[F, K, V](store)
 }
 
+final case class KeyValueTypedStoreInconsistencyError(message: String) extends Exception(message)
+
 final class KeyValueTypedStoreOps[F[_], K, V](
     // KeyValueTypedStore extensions / syntax
     private val store: KeyValueTypedStore[F, K, V]
 ) extends AnyVal {
   def get(key: K)(implicit f: Functor[F]): F[Option[V]] = store.get(Seq(key)).map(_.head)
 
+  def getUnsafe(key: K)(
+      implicit f: Sync[F],
+      line: sourcecode.Line,
+      file: sourcecode.File,
+      enclosing: sourcecode.Enclosing
+  ): F[V] = {
+    def source = s"${file.value}:${line.value} ${enclosing.value}"
+    def errMsg = s"KeyValueTypedStore is missing key ${key.toString}\n  $source"
+    get(key) >>= (_.liftTo(KeyValueTypedStoreInconsistencyError(errMsg)))
+  }
```

Comment:
As you noticed, `K` here will not be printed correctly because it's not serialized first.

Please create another instance of this extension in block-storage project with `K`  replaced with concrete type. This is a good example how it's easy to add more specific version of a function in the project where generic type is known.